src/cards/chase.js

```diff
@@ -0,0 +1,42 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+
+const CHASE_ALL_CARDS = 'https://creditcards.chase.com/all-credit-cards';
+const CHASE_DOMAIN = "https://creditcards.chase.com";
+
+class AllChaseCards {
+
+    async getBody() {
```

Comment:
NIT: I'd rename this function to something like getCards()

Reply:
yeah, u rite. copy/pasta typo.